components/Settings.js

```diff
@@ -202,94 +202,114 @@ const MenuButton = React.memo(({ name, select, selected }) => {
   )
 })
 
-/*const Presets = React.memo(({ show, presets, toggle, create, remove }) => {
+const Presets = React.memo(({ show, presets, toggle, create, remove, update }) => {
   return (
     <div className="settings-presets">
       <div className="settings-presets-header">
         <span>Presets</span>
-        {show && <button className="settings-presets-create" onClick={create}>create +</button>}
+        {false &&
+          show && (
+            <button className="settings-presets-create" onClick={create}>
+              create +
+            </button>
+          )}
         <button className="settings-presets-arrow" onClick={toggle}>
           {show ? <Arrows.Up /> : <Arrows.Down />}
         </button>
       </div>
       {show && (
         <div className="settings-presets-content">
-          {presets.map(({ id, backgroundColor, userCreated }) => (
-            <div key={id} className="settings-presets-preset" style={{
-              backgroundColor
-            }}>
-              {
-                userCreated ? <button className="settings-presets-remove" onClick={() => remove(id)}><Remove /></button> : null
-              }
-            </div>
+          {presets.map(preset => (
+            <button
+              key={preset.name}
+              className="settings-presets-preset"
+              style={{ backgroundColor: preset.backgroundColor }}
+              onClick={() => update(preset)}
+            >
+              {preset.custom ? (
+                <button className="settings-presets-remove" onClick={() => remove(preset.name)}>
```

Comment:
Can you hook up `remove` to remove that preset from the Editor's state? (:

Reply:
Yeah I was going to wait until we had custom presets to remove, but I'll do it now